src/OpenLoco/src/Town.cpp

```diff
@@ -25,6 +26,9 @@ namespace OpenLoco
     {
         recalculateSize();
 
+        if (Config::get().townGrowthDisabled)
```

Comment:
have you done checks to make sure this is purely grow after this point?

Reply:
If returned early here, no new buildings are built, nor are existing buildings demolished to make way for newer ones. Existing buildings that are under construction are still finishing construction over time, however — which is expected, as that's a different update function.

I tested this by leaving the game (Boulder Breakers) running for a long time in-game. Once the new option is unchecked, expansion resumes again.

However, I haven't inspected the `grow` function in-depth. I am going by observation only.